packages/common/src/utils/SafeLocalStorage.ts

```diff
@@ -37,7 +37,7 @@ export const SafeLocalStorage = {
     key: Key,
     value: SafeLocalStorageItems[Key]
   ): void {
-    if (isSafe()) {
+    if (isSafe() && value !== undefined) {
```

Comment:
```suggestion
    if (value && isSafe()) {
```

Reply:
Should not happen as they are strings but this might result in a weird behavior if `value` is `false`